src/channel.cr

```diff
@@ -127,61 +151,71 @@ abstract class Channel(T)
     ReceiveAction.new(self)
   end
 
-  # :nodoc:
```

Comment:
Docs comments on these are fine, but these look like implementation details that should retain the `:nodoc:` flag?

Reply:
I don't think these structs are part of the public API. Nor should they. Adding comments is great, but please keep `:nodoc:`.

Reply:
I remove it to add a comment, and I forget to put it again.

Reply:
Don't document the Action structs unless you're writing actual implementation docs that will help developers to understand the internal API design.